PSModule/AlmLabMgmtClient/SDK/Handler/PollHandler.cs

```diff
@@ -0,0 +1,93 @@
+using PSModule.AlmLabMgmtClient.SDK.Interface;
+using PSModule.AlmLabMgmtClient.SDK.Util;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace PSModule.AlmLabMgmtClient.SDK.Handler
+{
+    public abstract class PollHandler : HandlerBase
+    {
+        private readonly int _interval = 5000; // millisecond
+        protected PollHandler(IClient client, string entityId) : base(client, entityId)
+        {
+        }
+
+        protected PollHandler(IClient client, string entityId, int interval) : base(client, entityId)
+        {
+            _interval = interval;

```

Comment:
Need to check if the `interval` is valid.
https://docs.microsoft.com/en-us/dotnet/api/system.threading.thread.sleep?view=net-5.0#System_Threading_Thread_Sleep_System_Int32_